DependencyInjection/Configuration.php

```diff
@@ -158,6 +159,7 @@ public function getConfigTreeBuilder()
                                                 ->scalarNode('admin')->end()
                                                 ->scalarNode('label')->end()
                                                 ->scalarNode('route')->end()
+                                                ->booleanNode('route_absolute')->defaultTrue()->end()
```

Comment:
please add a description with `->info()` here and add this new config option to the docs.